testset_maker/init_testset.py

```diff
@@ -0,0 +1,30 @@
+import csv
```

Comment:
I think we can remove this functionality. This can be done by the interactive dataset creator we already have and there's more succinct ways to do it with pandas dataframes.